utils/ioutils/questionnaire.go

```diff
@@ -82,7 +84,9 @@ func interruptKeyBind() prompt.Option {
 	interrupt := prompt.KeyBind{
 		Key: prompt.ControlC,
 		Fn: func(buf *prompt.Buffer) {
-			panic("Interrupted")
+			// Gracefully exit the program
+			fmt.Println("\nOperation interrupted. Exiting...")
+			os.Exit(0)
```

Comment:
While this change might offer a better user experience (UX), please consider that it could prevent caller methods from running their deferred scopes. This might leave some resources open, which could ultimately result in Zombie handles in Windows.
Consider using panic + recover.